mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
+warn_return_any = True
+warn_unused_configs = True
+exclude = (integration|invoke/vendor|site|tests|tasks)
```

Comment:
For a more comprehensive check, I'd suggest copying this config: https://github.com/aio-libs/aiohttp-jinja2/blob/master/.mypy.ini

Then comment out any options that introduce too many errors, with the idea that they could be introduced in later PRs iteratively.

Reply:
There were actually two configs due to that rebase `mypy.ini` and `pyproject.toml`. I reconfigured everything to `pyproject.toml`. As for the additional configs I have no problem with it but I think it's best to get @bitprophet feedback so I'll add it commented out for now.